src/renderer/upscaling_utils.hpp

```diff
@@ -31,23 +31,23 @@ namespace rigel::renderer
 
 class Renderer;
 
-struct ViewPortInfo
+struct viewportInfo
 {
   base::Vec2 mOffset;
   base::Size<int> mSize;
   base::Vec2f mScale;
 };
 
 
-struct WidescreenViewPortInfo
+struct WidescreenviewportInfo
```

Comment:
Here as well, the correct name would be `WidescreenViewportInfo`. I mentioned this already in my last response.